libsolidity/codegen/ir/IRGenerationContext.cpp

```diff
@@ -173,10 +162,10 @@ string IRGenerationContext::generateInternalDispatchFunction(Arity const& _arity
 		)");
 		templ("functionName", funName);
 		templ("comma", _arity.in > 0 ? "," : "");
-		templ("in", suffixedVariableNameList("in_", 0, _arity.in));
+		templ("in", IRNames::internalFunctionInputLocals(_arity));
```

Comment:
I'm not sure we should create functions for all possible strings. Can this one for example clash with something else at all?

Reply:
I don't think so. I just thought that it would not hurt to have them all together.

I'll move them back.

Reply:
Done.